db/seeds.rb

```diff
@@ -24,7 +24,32 @@
 	description: "my description of Futurama",
 	small_cover_url: "/tmp/futurama.jpg",
 	large_cover_url: "/tmp/futurama.jpg",
-	category_id: 3)
+	category_id: 2)
```

Comment:
Hi Brandon,
I gave this a shot, category:  comedy for each of the seed records.
I saved the seeds.rb
Ran rake db:reset db:test:prepare
When I start up rails s, I get this:
Started GET "/home" for 127.0.0.1 at 2015-02-20 08:15:57 -0600
Processing by VideosController#index as HTML
  User Load (0.4ms)  SELECT  "users".* FROM "users"  WHERE "users"."id" = $1 LIMIT 1  [["id", 1]]
Completed 404 Not Found in 77ms

ActiveRecord::RecordNotFound - Couldn't find User with 'id'=1:

I'm not sure how to go forward.